controller/namedspaces_test.go

```diff
@@ -146,3 +149,115 @@ func (rest *TestNamedSpaceREST) TestShow() {
 		assert.Equal(t, &fxt.Spaces[0].Description, namedspace.Data.Attributes.Description)
 	})
 }
+
+func (rest *TestNamedSpaceREST) TestSuccessSortListSpaces() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
+
+	// names of the spaces to be created
+	inputSpaces := []string{
+		"Test 24",
+		"Test 22",
+		"Test 23",
+	}
+	// randominze the space names so that they don't clash
+	random := testsupport.CreateRandomValidTestName("")
+	for i := range inputSpaces {
+		inputSpaces[i] += random
+	}
+
+	// create actual spaces with those names
+	fxt := tf.NewTestFixture(t, rest.DB, tf.Spaces(len(inputSpaces),
+		func(fxt *tf.TestFixture, idx int) error {
+			fxt.Spaces[idx].Name = inputSpaces[idx]
+			return nil
+		}),
+	)
+	identity := *fxt.Identities[0]
+
+	// Now try to list the data in various circumstances
+	tests := []struct {
```

Comment:
Please also add a test for a bad sort parameter like `""` or `"nonesense"` here.